packages/fe-fpm-writer/test/unit/column.test.ts

```diff
@@ -150,6 +150,66 @@ describe('CustomAction', () => {
             expect(testFS.read(expectedFragmentPath)).toMatchSnapshot();
         });
 
+        test('version 1.102, no handler, filename lowercase', () => {
+            const testCustomColumn: CustomTableColumn = {
+                ...customColumn,
+                name: 'newCustomColumn',
+                folder: 'extensions/custom'
+            };
+
+            const testFS = generateCustomColumn(testDir, { ...testCustomColumn, minUI5Version: '1.102' });
+            const updatedManifest: any = testFS.readJSON(join(testDir, 'webapp/manifest.json'));
+
+            const settings = updatedManifest['sap.ui5']['routing']['targets']['sample']['options']['settings'];
```

Comment:
Same comment as before, do we really need to test the snapshots here? This test is about the correct naming, so let's focus on that in the test.

Reply:
@tobiasqueck Sorry, I don't get the point. I'm testing the outcome of the function call, same as the other tests are doing. Should be the mem-fs content.

Reply:
@stefanschreck let me try again. The purpose of this bug fix is that the filename patter (lower/upper case) is not enforced anymore. We already have sufficient tests to check that the generated outcome (content of file) is correct. What we asked you to add, was a test verifying the changed functionality. Technically, you could also do a snapshot test of the whole webapp folder but you don't because it would be harder to maintain and review. 
So, please focus in your additional test on the functionality that actually needs to be tested, and not functionality that is already covered in other tests.

Reply:
Ok - as the filename is just part of the  file path, a simple existence check will be sufficint.